core/src/main/java/io/questdb/cutlass/text/ParallelCsvFileImporter.java

```diff
@@ -518,7 +521,7 @@ private void attachPartitions(TableWriter writer) throws TextImportException {
             final CharSequence partitionDirName = partition.name;
             try {
                 final long timestamp = PartitionBy.parsePartitionDirName(partitionDirName, partitionBy);
-                writer.attachPartition(timestamp, false);
+                writer.attachPartition(timestamp, partition.importedRows);
```

Comment:
will this make import slower? Shall we add an argument to make sure attach goes via shortest path possible? What jumps at me, is that if "copy on attach" is enabled, this will incur extra unnecessary cost

Reply:
This makes import faster
I'll add condition to ignore copy on attach for imports